File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/CommonDialogs/OpenFileDialog/Features/TitlePage.vb
Namespace CompanionKit.Controls.CommonDialogs.OpenFileDialog.Features

	Public Class TitlePage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

		End Sub

       ''' <summary>
        ''' Handles Load event of example UserControl.
        ''' Sets initial title of the open file dialog.
        ''' </summary>
        Private Sub TitlePage_Load(sender As Object, e As EventArgs) Handles Me.Load
            ' Set initial title of the open file dialog.
            Me.mobjTitleFileDialogTextBox.Text = Me.mobjDemoOpenFileDialog.Title
        End Sub

        ''' <summary>
        ''' Handles Click event of the Button.
        ''' Sets specified title for open file dialog and open it.
        ''' </summary>
        Private Sub mobjOpenFileDialogButton_Click(sender As Object, e As EventArgs) Handles mobjOpenFileDialogButton.Click
            ' Set specified title for open file dialog and open it.
            Me.mobjDemoOpenFileDialog.Title = Me.mobjTitleFileDialogTextBox.Text
            Me.mobjDemoOpenFileDialog.ShowDialog()
        End Sub
    End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/CommonDialogs/OpenFileDialog/Features/TitlePage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.CommonDialogs.OpenFileDialog.Features
{
    public partial class TitlePage : UserControl
    {
        public TitlePage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles Load event of example UserControl.
        /// Sets initial title of the open file dialog.
        /// </summary>
        private void TitlePage_Load(object sender, EventArgs e)
        {
            // Set initial title of the open file dialog.
            this.mobjTitleFileDialogTextBox.Text = this.mobjDemoOpenFileDialog.Title;
        }

        /// <summary>
        /// Handles Click event of the Button.
        /// Sets specified title for open file dialog and open it.
        /// </summary>
        private void mobjOpenFileDialogButton_Click(object sender, EventArgs e)
        {
            // Set specified title for open file dialog and open it.
            this.mobjDemoOpenFileDialog.Title = this.mobjTitleFileDialogTextBox.Text;
            this.mobjDemoOpenFileDialog.ShowDialog();
        }
    }
}